llvm/include/llvm/Transforms/Instrumentation/AddressSanitizerCommon.h

```diff
@@ -58,6 +58,13 @@ void getAddressSanitizerParams(const Triple &TargetTriple, int LongSize,
                                bool IsKasan, uint64_t *ShadowBase,
                                int *MappingScale, bool *OrShadowOffset);
 
+/// Remove memory attributes that are incompatible with the instrumentation
+/// added by AddressSanitizer and HWSanitizer. See more details in the
+/// implementation.
+/// \p ReadsArgMem - whether to remove memory attributes from function
```

Comment:
The comment doesn't really match the argument name, should this either by RemoveWriteOnly to match the description or the description updated, e.g. something like `indicates if the function's arguments may be read by instrumentation and require removing `writeonly` attributes